transport/src/main/java/io/netty/channel/embedded/EmbeddedChannelId.java

```diff
@@ -40,7 +40,7 @@ public String asLongText() {
     }
 
     @Override
-    public int compareTo(ChannelId o) {
+    public int compareTo(final ChannelId o) {
```

Comment:
I think it's always good to clean the code up while making modifications as long as it doesn't drain too much cognitive load. :-)